### PR TITLE
Tickets/dm 31945

### PR DESCRIPTION
**Pull request by MelissaGraham**
https://jira.lsstcorp.org/browse/DM-31945

### CODE REVIEW

File: DMTN-049.tex [hunk 1]
```diff
@@ -246,11 +246,12 @@ \subsection{Scientific Utility}
 The development of such advanced photo-$z$ algorithms is an active research topic within the Dark Energy Science Collaboration \citep{2018arXiv180901669T}, and is a significant effort which the Rubin Observatory staff should not attempt to replicate. 
 \item \textbf{Serve Diverse Science Needs -- }
 The selected photo-$z$ estimator(s) should serve as wide a variety of scientists as possible, especially those who would or could not generate custom photo-$z$ for themselves (Appendix \ref{sec:use}). 
-Photo-$z$ estimators that can return reliable results not just for galaxies but also for, e.g., AGN and quasars, and/or that can also classify stellar types, should be considered favorably.
+At least one photo-$z$ estimators that can return reliable results not just for galaxies but also for AGN (and, e.g., quasars) should be selected.
```
Comment (TallJimbo) [inline DMTN-049.tex]:
```suggestion
At least one photo-$z$ estimator that can return reliable results not just for galaxies but also for AGN (and, e.g., quasars) should be selected.
```

File: DMTN-049.tex [hunk 2]
```diff
@@ -275,118 +276,99 @@ \subsection{Scientific Performance}
...
-\textbf{LOR Purpose:}
-The Rubin science community has a considerable wealth of expertise in generating PZ catalogs and will be the primary users of the Object PZ data products.
-These letters provide a formal opportunity for the science community to advocate for one or more photo-$z$ estimators that will meet their minimum scientific needs, and/or to define these minimum needs so that DM can consider them when shortlisting PZ estimators.
+\subsection{Summary of the Letters Submitted (and DM's Shortlist)} \label{ssec:lor_choice}
 
-\textbf{LOR Writers:}
-Any group or individual who would use the LSST Object PZ for their future scientific analyses, and/or are developers of potentially suitable PZ estimators, are encouraged to submit an letter.
-The DM team is especially interested in hearing the needs of scientists who plan to use the LSST Object catalog but who would/could not generate custom PZ estimates.
-LORs are not restricted to Rubin data rights holders.
+The Rubin Observatory Data Management (DM) team sincerely thank everyone who engaged with the PZ LOR process.
```
Comment (TallJimbo) [inline DMTN-049.tex]:
```suggestion
The Rubin Observatory Data Management (DM) team sincerely thanks everyone who engaged with the PZ LOR process.
```

File: DMTN-049.tex [hunk 3]
```diff
@@ -275,118 +276,99 @@ \subsection{Scientific Performance}
...
+\begin{itemize}
+\item assembling training sets and validation metrics,
+\item implementing photo-$z$ estimators with the LSST Science Pipelines,
+\item using the data previews to validate the estimators' performance, and
+\item documenting, evaluating, and sharing the results.
+\end{itemize}
 
-Types of commissioning data that might be most useful for scientific validation of photo-$z$ estimators are described in \S~\ref{ssec:pzcoop_commissioning}.
-A preliminary collection of photo-$z$ validation techniques based are presented in Appendix \ref{sec:imp}, and a preliminary collection of training and calibration data in Appendix \ref{sec:dp}.
+These activities will inform DM's selection of photo-$z$ estimator(s) for the Object catalog, but more broadly would also enable the science community to prepare for photo-$z$ science with the LSST and the to develop user-generated photo-$z$ catalogs.
```
Comment (TallJimbo) [inline DMTN-049.tex]:
```suggestion
These activities will inform DM's selection of photo-$z$ estimator(s) for the Object catalog, but more broadly would also enable the science community to prepare for photo-$z$ science with the LSST and to develop user-generated photo-$z$ catalogs.
```

File: DMTN-049.tex [hunk 4]
```diff
@@ -666,4 +648,117 @@ \subsection{Considerations for Maximizing Early Science}\label{ssec:use_LOY1}
...
+\item make all PZ-related code, inputs, metadata, etc. available (for reproducibility)
+\item correct for MW dust before passing fluxes to the PZ estimator
+\item provide users with a framework and tools to cross-match to non-LSST catalogs
+\end{itemize}
+Any of these recommendations which were not already part of the evaluation criteria in \S~\ref{sec:eval} have been added.
+
+Recommendations that present a challenge:
+\begin{itemize}
+\item incorporate non-LSST, non-optical photometry into the PZ estimates
+\item cross-match the LSST Objects catalogs with external data sets (e.g., an X-ray AGN catalog) and provide the results with flags and identifiers
```
Comment (TallJimbo) [inline DMTN-049.tex]:
```suggestion
\item cross-match the LSST Object catalogs with external data sets (e.g., an X-ray AGN catalog) and provide the results with flags and identifiers
```

File: DMTN-049.tex [hunk 5]
```diff
@@ -666,4 +648,117 @@ \subsection{Considerations for Maximizing Early Science}\label{ssec:use_LOY1}
...
+\item in the alert packets, provide host galaxy Object data (PZ, photometry, shape) and non-LSST survey data for host galaxies (spec-z, PZ, photometry)
+\end{itemize}
+These challenges are out of scope for the Rubin Construction project (and some violate the current Rubin data rights policy), and some items (such as providing cross-matched catalogs) are best left to the expertise of the science community.
+However, just because an idea or data product presents a challenge, or is beyond the scope of Rubin construction and commissioning, does not mean it could never be accomplished.
+Adopting and federating value-added user-generated data products, and updating algorithms in the LSST Science Pipelines, is an Operations-era (2024 and beyond) activity that will take input from the science community via the Rubin Users Committee, the Community Engagement Team, the Science Advisory Committee, and other such groups.
+It is very useful to have these suggestions raised early on so that the DM team can help the science community prepare for Operations, all of the LOR providing these recommendations are much appreciated.
```
Comment (TallJimbo) [inline DMTN-049.tex]:
```suggestion
It is very useful to have these suggestions raised early on so that the DM team can help the science community prepare for Operations; all of the LOR providing these recommendations are much appreciated.
```

**Review by TallJimbo — APPROVED** [review]
Looks good to me - all of my PR comments are tiny style/grammar things.

One general formatting issue: unless LaTeX technotes have some secret sauce I'm not aware of, there are a lot of plain double-quotes that won't render properly.

**Comment by MelissaGraham** [comment]
> One general formatting issue: unless LaTeX technotes have some secret sauce I'm not aware of, there are a lot of plain double-quotes that won't render properly.

They render ok but they do kind of point backwards -- will fix.

File: DMTN-049.tex [hunk 6]
```diff
@@ -275,117 +276,126 @@ \subsection{Scientific Performance}
...
 
-\textit{TBD: This section will summarize the "Letters of Recommendation" and describe which community-vetted photo-$z$ estimator(s) have been shortlisted for the photo-$z$ validation cooperative, and why.}
+It is expected that \textit{most} of the science community participating in the ``Photo-$z$ Validation Cooperative" will use the commissioning data that is released as part of Data Preview 1 and 2 (DP1 and DP2), and will not need to formally join the Rubin Commissioning Team and have access to the unreleased commissioning data.
 
-\section{Photo-$z$ Validation Cooperative}\label{sec:pzcoop}
+However, as mentioned above the DM team is very limited in terms of the support it can provide for implementation, and furthermore there are some aspects of implementing and validating photo-$z$ estimators which would benefit from a connection with the unreleased commissioning data.
 
-\textbf{This preliminary draft description of this proposed cooperative effort will be updated based on community input.}
+Thus, Rubin Observatory is seeking to identify a small group of $\sim$5-10 individuals who have expertise with photo-$z$ estimators to join the Rubin Commissioning Team.
```
Comment (leannep) [inline DMTN-049.tex:368]: I suspect most will want to join to get early access to data.  Would we attempt to downselect to a 'small group' if everyone asks to join? tempted to rephrase this to something like "any science community participating in the ``Photo-$z$ Validation Cooperative" who wishes to join the Rubin Commissioning team is invited to do so"

File: DMTN-049.tex [hunk 7]
```diff
@@ -275,117 +276,126 @@ \subsection{Scientific Performance}
...
 
-\textbf{Draft:} \\
-As the scientific community has a considerable wealth of expertise in generating photo-z catalogs, and will be the primary users of the photo-z data product, they are the best suited to scientifically validate the {\tt Object} catalog photo-$z$.
-Towards this end the DM team plans to facilitate a "Photo-$z$ Validation Cooperative" to analyze photo-$z$ estimates based on LSST commissioning data and share results.
-This is intended to be a collaborative endeavor between project and community to maximize the scientific utility of the {\tt Object} catalog photo-$z$ data products.
+In exchange for access to the unreleased commissioning data and Commissioning Team membership, these individuals would commit to implementing the shortlisted PZ estimators and to leading the science community's validation efforts during DP1 and DP2.
```
Comment (leannep) [inline DMTN-049.tex]: This sounds a bit quid pro quo. How about rephrasing to say something like "as a member of the Rubin Commissioning team, these individuals would be responsible for implementing the shortlisted ...." which is more in keeping with the manner in which all other community Commissioning team members will work.

File: DMTN-049.tex [hunk 8]
```diff
@@ -656,14 +668,127 @@ \subsection{Considerations for Maximizing Early Science}\label{ssec:use_LOY1}
 Given the uncertainties on the actual construction schedule and commissioning period, several different plans for early science are being considered.
```
Comment (leannep) [inline DMTN-049.tex]: Add in a referecen here to the new Early Science document. rtn-011.lsst.io

**Review by leannep — CHANGES_REQUESTED** [review]
Looks good - a few comments.

File: DMTN-049.tex (hunk 6 above)
Comment (MelissaGraham) [reply DMTN-049.tex:368]: Re. downselecting, my impression is that a team of 5-10 is appropriate and matches other Commissioning AO teams, and yes we should downselect if this opportunity is oversubscribed. The "PZ-related" Commissioning team members should be limited to only people who are committing time and effort to implementation, and their statements must be reviewed like the other Commissioning AO teams. Also, the 5-10 number is probably appropriate because DM is limited in how many people it can help with PZ implementation. All this comes from talking with Keith and Jim about how these PZ-related Commissioning Team members would work. 

Re. "anyone can participate", later on we do say that anyone interested in committing to Commissioning Team participation can reach out (to me) to put together a statement. This is not limited to PZ LOR writers nor the shortlisted estimator teams.

File: DMTN-049.tex (hunk 7 above)
Comment (MelissaGraham) [reply DMTN-049.tex]: Totally, changed to: "As a member of the Rubin Commissioning team, these individuals would be responsible for implementing the shortlisted photo-$z$ estimators and leading the science community's validation efforts during DP1 and DP2."

File: DMTN-049.tex (hunk 8 above)
Comment (MelissaGraham) [reply DMTN-049.tex]: RTN-011 was being referenced in the previous sentence, so I moved it to this sentence, after "considered".

File: DMTN-049.tex (hunk 6 above)
Comment (leannep) [reply DMTN-049.tex:368]: OK, That all makes sense and I agree with all the points.

**Review by leannep — APPROVED** [review]
